### DISCUSSION TITLE
did you mean to import ohash/dist/index.cjs

### DISCUSSION THREAD

**Discussion by nigiwen** [nuxt/nuxt#20700] [Questions]
# did not use ohash in nuxt3
![img_v2_4d820347-0fce-4c4e-b8da-5dcdb2cffd3g](https://user-images.githubusercontent.com/67454874/236591102-09404ad8-e3df-46c1-a3ef-08bd7d549555.jpg)

**Comment by logotip4ik** [comment]
Have you tried removing the lock file and reinstalling dependencies ?

**Reply by nigiwen** [reply]
yes, but not good

**Comment by nigiwen** [comment]
# Very bad, my other project also went down
![img_v2_d7227d7d-34d8-41fd-9c77-536a5dbea31g](https://user-images.githubusercontent.com/67454874/236717171-493fe6ea-cca6-496e-88fb-82bfccbcd7a9.jpg)
